src/components/app-core/forms.css

```diff
@@ -41,15 +41,16 @@ select {
 input:focus,
 select:focus,
 textarea:focus {
-  margin-bottom: -1px; /* Decrease the margin-bottom, to prevent jump caused by enlarged border */
-  border-bottom-width: 2px;
+  margin-bottom: -2px; /* Decrease the margin-bottom, to prevent jump caused by enlarged border */
+  margin-right: -2px; /* Decrease the margin-right, to prevent jump caused by enlarged border */
+  border-width: 2px;
```

Comment:
The border still jumps a bit. I think nowadays we can better use `outline: 2px solid var(--html-blue)` on focus, so we can get rid of the margin hacks. And then we probably also have to set `outline-color: var(--soft-red)` on the invalid styling.

Reply:
Yes perfect!